src/dvclive/live.py

```diff
@@ -121,6 +121,9 @@ def _init_cleanup(self):
             if f and os.path.exists(f):
                 os.remove(f)
 
+        if self.dvc_file and os.path.exists(self.dvc_file):
```

Comment:
I remember we had a discussion the first time we added the make_dvcyaml about not wanting to remove it on cleanup https://github.com/iterative/dvclive/pull/366#discussion_r1036232488

Do you think the concerns no longer apply (the config for sklearn is now auto-generated) and we can consider the dvclive/dvc.yaml not modifiable?

Reply:
It looks like we are overriding anyhow in the code, so we never preserved the existing dvcyaml 😅